content/controllers.md

```diff
@@ -268,6 +268,8 @@ getDocs(@Query('version') version) {
 
 Routes with static paths won't work when you need to accept **dynamic data** as part of the request (e.g., `GET /cats/1` to get cat with id `1`). In order to define routes with parameters, we can add route parameter **tokens** in the path of the route to capture the dynamic value at that position in the request URL. The route parameter token in the `@Get()` decorator example below demonstrates this usage. Route parameters declared in this way can be accessed using the `@Param()` decorator, which should be added to the method signature.
 
+> info **Warning** Routes with parameters should be defined after any static routes. This prevents the parameterized routes from intercepting traffic destined for the static routes.
```

Comment:
```suggestion
> info **Warning** Routes with parameters should be defined after any static paths. This prevents the parameterized routes from intercepting traffic destined for the static paths.
```

as this page uses 'static paths' everywhere, I think keeping the same word sounds less confusing